File: array/268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int totalSum = nums.length * (nums.length + 1) /2;
        int giveSum = 0;
        for(int i=0; i<nums.length; i++){
            giveSum += nums[i];
        }
        return totalSum - giveSum;
    }
}

File: array/268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        for(int i=0; i<=nums.size(); i++){
            int count = 0;
            for(int j=0; j<nums.size(); j++){
                if(i==nums[j]){
                    count++;
                    break;
                }
            }
            if(count!=1){
                return i;
            }
        }
        return -1;
    }
};